modules/installing-gcp-user-defined-labels-and-tags.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+// * installing/installing_gcp/installing-gcp-customizations.adoc
+
+:_content-type: CONCEPT
+[id="installing-gcp-user-defined-labels-and-tags_{context}"]
+= Configuring the user-defined labels and tags for GCP
+
+
+Google Cloud Platform (GCP) provides `labels` and `tags` that help to identify and organize the resources created for a specific {product-title} cluster, making them easier to manage. 
+
+You can define `labels` and `tags` for each GCP resource only during {product-title} cluster installation. 
+
+[IMPORTANT]
+====
+User-defined `labels` and `tags` are not supported for {product-title} clusters upgraded to {product-title} 4.14 version.
+====
+
+.User-defined labels
+
+User-defined `labels` and {product-title} specific labels are applied only to resources created by {product-title} installer and its core components such as: 
+* GCP filestore CSI driver operator
+* GCP PD CSI driver operator
+* Image registry operator
+* Machine API provider for GCP
+
+
+The user-defined `labels` and {product-title} specific labels are not applied on the resources created by any other operators or the kubernetes in-tree components that create resources, for example, the Ingress load balancers.
+
+The following are the requirements for the user-defined labels:
+
+* A label key and value must have a minimum of 1 character and can have a maximum of 63 characters.
+* A label key and value must contain only lowercase letters, numeric characters, underscore (`_`), and dash (`-`).
+* A label key must start with a lowercase letter.
+* You can configure a maximum of 32 labels per resource. Each resource can have a maximum of 64 labels, and 32 labels are reserved for internal use by {product-title}.
+
+.Limitations for user-defined labels
+
+* Labels for `ComputeAddress` are supported in the GCP beta version. {product-title} does not add labels to the resource.
+
+User-defined labels and {product-title} labels are available on the following GCP resources:
+
+* Compute disk
+* Compute instance
+* Compute image
+* Compute forwarding rule
+* DNS managed zone
+* Filestore instance
+* Storage bucket
+
+.User-defined tags
+
+User-defined `tags` are attached to resources created by {product-title} core operator cluster image registry operator and not on the resources created by any other operators or the kubernetes in-tree components. In {product-title} version {product-version}, resources created by the foloowing operators are not tagged:
+
+* GCP filestore CSI driver operator
+* GCP PD CSI driver operator
+* Installer operator
```

Comment:
```suggestion
* Installer
```

Reply:
Ack.

Reply:
As it was part of the redundant text, removed.